.github/workflows/release-go.yaml

```diff
@@ -25,14 +25,14 @@ jobs:
         env:
           QUAYIO_ORG: quay.io/numaio
           PLATFORM: linux/x86_64
-          TARGET: numalogic-prometheus/prometheus-serde
+          TARGET: numalogic-prometheus/udf/prometheus-serde
```

Comment:
why do we need `udf` ?

Reply:
Currently, numalogic-prometheus is pushing to `numalogic-prometheus/udf` repo in quay.io
just following the same